CENSUS_METADATA.md

```diff
@@ -0,0 +1,778 @@
+is not included in the `Variable` object. As noted above, classifications for geographic variables are automatically
+created and this is reflected in the values in this object.
+
+The `Parent_Classification_Mnemonic`, `Signed_Off_Flag`, `Flat_Classification_Flag` and `Id` field in `Classification.csv` are ignored.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `Mnemonic_2011` | `String` | `Classification.Mnemonic_2011` or `null` for geographic variables | |
+| `Default_Classification_Flag` | `String` | `Classification.Default_Classification_Flag` or `null` for geographic variables | |
+| `Version` | `String!` | `Classification.Version` or `Variable.Version` for geographic variables | |
+| `ONS_Variable` | `ONS_Variable!` | Keyed on `Classification.Variable_Mnemonic` or `Variable.Variable_Mnemonic` for geographic variables | |
+| `Topics` | `[Topic]!` | Keyed on `Topic_Classification.Classification_Mnemonic`/`Topic_Mnemonic` or `[]` for geographic variables | |
+
+## ONS_Variable
+
+`ONS_Variable` is a user defined object.
+
+The data is sourced from `Variable.csv`.
+
+The `Id`, `Signed_Off_Flag` and `Number_Of_Classifications` fields in `Variable.csv` are ignored.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `Variable_Mnemonic` | `String!` | `Variable.Variable_Mnemonic` | |
+| `Variable_Title` | `String!` | `Variable.Variable_Title` | `Variable.Variable_Title_Welsh`|
+| `Variable_Mnemonic_2011` | `String` | `Variable.Variable_Mnemonic_2011` | |
+| `Comparability_Comments` | `String` | `Variable.Comparability_Comments` | `Variable.Comparability_Comments_Welsh` |
+| `Uk_Comparison_Comments` | `String` | `Variable.Uk_Comparison_Comments` | `Variable.Uk_Comparison_Comments_Welsh` |
+| `Geographic_Abbreviation` | `String` | `Variable.Geographic_Abbreviation` | `Variable.Geographic_Abbreviation_Welsh` |
+| `Geographic_Theme` | `String` | `Variable.Geographic_Theme` | `Variable.Geographic_Theme_Welsh` |
+| `Geographic_Coverage` | `String` | `Variable.Geographic_Coverage` | `Variable.Geographic_Coverage_Welsh` |
+| `Version` | `String!` | `Variable.Version` | |
+| `Statistical_Unit` | `Statistical_Unit` | Keyed on `Variable.Statistical_Unit` | |
+| `Keywords` | `[String]!` | List of `Variable_Keyword.Variable_Keyword` values keyed on `Variable_Keyword.Variable_Mnemonic` | List of `Variable_Keyword.Variable_Keyword_Welsh` values keyed on `Variable_Keyword.Variable_Mnemonic`|
+| `Topic` | `Topic` | Keyed on `Variable.Topic_Mnemonic` | |
+| `Questions` | `[Question]!` | Keyed on `Variable.Source_Question.Variable_Mnemonic`/`Source_Question_Code` | |
+| `Variable_Type` | `Variable_Type!` | `Variable.Variable_Type_Code` | |
+| `Quality_Statement_Text` | `String` | `Variable.Quality_Statement_Text` | |
+| `Quality_Summary_URL` | `String` | `Variable.Quality_Summary_URL` | |
+
+## Variable_Type
```

Comment:
Should be `###`?

Reply:
I'm finding the heading sizes hard to follow, are they supposed to all be the same size or is there some purposeful size change happening that I'm missing?

Reply:
All those object types should be `##`. There appears to be a mix of `##` and `###`. Will change.